apps/token/src/routes/governance/components/proposal-votes-table/proposal-votes-table.tsx

```diff
@@ -44,20 +59,44 @@ export const ProposalVotesTable = ({ proposal }: ProposalVotesTableProps) => {
     >
       <KeyValueTableRow>
         {t('willPass')}
-        {willPass ? '👍' : '👎'}
+        {isUpdateMarket
```

Comment:
suggestion (non blocking): possibly add a Thumbs component? Should make this ui a touch more maintanable as well as make this nested ternary a bit more readable..

This nested ternary could then be replaced with something more readable


something like:
```
isUpdateMarket ? (
  <Thumbs up={updateMarketWillPass} upText={updateMarketVotePassMethod} />
) : (
  <Thumbs up={willPassByTokenVote}
)
```

Reply:
Nice idea!